src/esp/assets/BaseMesh.h

```diff
@@ -3,6 +3,12 @@
 // LICENSE file in the root directory of this source tree.
 
 #pragma once
+
+/** @file
+ * @brief enum @ref esp::assets::SupportedMeshType, Class @ref
+ * esp::assets::BaseMesh
```

Comment:
Just a heads-up -- I remember Doxygen having strange issues with `@brief` that's on multiple lines. Maybe that isn't the case anymore and it got fixed, but be sure to check for suspicious-looking output ;)

Reply:
Thanks, I'll look out for this. Seems ok in this case.